android-components/components/browser/state/src/main/java/mozilla/components/browser/state/action/BrowserAction.kt

```diff
@@ -1334,6 +1334,11 @@ sealed class HistoryMetadataAction : BrowserAction() {
  * [BrowserAction] implementations related to updating search engines in [SearchState].
  */
 sealed class SearchAction : BrowserAction() {
+    /**
+     * Refreshes the list of search engines when language is changed.
```

Comment:
```suggestion
     * Refreshes the list of search engines.
```
We should be a bit generic here since how the action is called is up to the consumer of the component.